src/modules/default-store-redirect/server.ts

```diff
@@ -0,0 +1,44 @@
+import appConfig from 'config'
+import { serverHooks } from '@vue-storefront/core/server/hooks'
+import { defaultStoreRedirectHooksExecutors } from './hooks'
+
+/**
+ * Redirect requests without a store code to the default (or first) store
+ */
+if (appConfig.storeViews.multistore && appConfig.defaultStoreCodeRedirect.enabled) {
+  serverHooks.afterApplicationInitialized(({ app }) => {
+    const blacklist = appConfig.defaultStoreCodeRedirect.blacklist
+    const storeCodes = appConfig.storeViews.mapStoreUrlsFor
+
+    const blacklistStr = blacklist.join('|')
+    const storeCodesStr = storeCodes.join('|')
+    const hasStoreCode = new RegExp(
+      `^((?!\\/(${storeCodesStr})(\\/|$))(?!\\/(${blacklistStr})$))\\/?.*((?<=\\.html)|(?<!\\.[a-zA-Z0-9]*))$`
+    )
+
+    app.get(hasStoreCode, (req, res) => {
```

Comment:
`blacklist` config should not exist. It's like a solution for a problem that we want to create here :thinking: We can move this check here https://github.com/DivanteLtd/vue-storefront/blob/master/core/scripts/server.ts#L167 and return from `app.get('*', (req, res, next) => {` if route is redirected. Or we can add middleware before that https://github.com/DivanteLtd/vue-storefront/blob/master/core/scripts/server.ts#L161 (it needs to be tested, but we should have in that point only routes of our app, not webpack files)

Reply:
@gibkigonzo Yes, the blacklist I originally copied over from the source I used as a starting point for this module. I also intended to make it a config field because we had a situation where we have some specific routes that needed to be non-localized. But after reviewing this, it didn't make sense to do this and we will be removing this need. I'll remove the blacklist here as well.

Are you suggesting I move the blacklist check into server.ts somewhere? I don't fully understand what your suggestion is.